code-challenges.test.js

```diff
@@ -18,31 +18,75 @@
 
 // a) Create a test with an expect statement using the variable provided.
 
-var colors1 = ["purple", "blue", "green", "yellow", "pink"]
+// var colors1 = ["purple", "blue", "green", "yellow", "pink"]
 // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+// var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
 
+// Describe a function "jumbleArray", it has input of array and output of an array of the first item removed and the rest scrambled.
+describe("jumbleArray", () => {
+  it("takes in an array, removes the first item from the array and shuffles the remaining content.", () => {
+    let colors1 = ["purple", "blue", "green", "yellow", "pink"]
+    let colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+    let output1 = jumbleArray(colors1)
+    let output2 = jumbleArray(colors2)
+    //Check that the output has the first string removed
+    //Check that the output is contained in the original array.
+    expect(colors1).toEqual(expect.arrayContaining(output1))
+    expect(colors1).toHaveLength(4)
+    expect(colors1).not.toContain("purple")
+    expect(colors2).toEqual(expect.arrayContaining(output2))
+    expect(colors2).toHaveLength(5)
+    expect(colors2).not.toContain("chartreuse")
```

Comment:
This is robust testing! Great job getting creative and using multiple expects to check your desired results.